Dfe.Academies.External.Web.UnitTest/Controllers/SchoolControllerTests.cs

```diff
@@ -45,8 +44,7 @@ public async Task SchoolController___ReturnSchoolDetailsPartialViewPopulated___R
 	    // arrange
 	    var mockLogger = new Mock<ILogger<SchoolController>>();
 	    var mockReferenceDataRetrievalService = new Mock<IReferenceDataRetrievalService>();
-	    var mockConversionApplicationRetrievalService = new Mock<IConversionApplicationRetrievalService>();
-        var schoolController = new SchoolController(mockLogger.Object, mockReferenceDataRetrievalService.Object, mockConversionApplicationRetrievalService.Object);
+        var schoolController = new SchoolController(mockLogger.Object, mockReferenceDataRetrievalService.Object);
```

Comment:
Can we have the test name:

```
ReturnSchoolDetailsPartialViewPopulated___[Scenario]___ReturnsPartialView
```

I'm not quite sure what the scenario is here, "SchoolSelected" maybe, as opposed to "SchoolNull"?

Reply:
Amended name in PR68 as that's where the school controller tests are contained